### DISCUSSION TITLE
infinite list of errors

### DISCUSSION THREAD

**Discussion by 4lt3r3go** [Klace/stable-diffusion-webui-images-browser#10] [Q&A]
i'm getting an infinite list of errors ""non png file in directory...."" when the extention scan the folders.
This is probably not supporting jpgs? 

_I dont use png at all since i havent noticed any visible difference between png and jpgs,
also exif are stored in the jpgs same as png so i switched to jpgs months ago (my harddrive thanks me)_

**Comment by younyokel** [comment]
Use https://github.com/AlUlkesh/stable-diffusion-webui-images-browser, it's more recent
